build.gradle

```diff
@@ -9,6 +9,10 @@ repositories {
 	}
 }
 
+runClient.doFirst {
+	args = ['--username', "Player${new Random().nextInt(1000)}", '--uuid', UUID.randomUUID()]
+}
+
```

Comment:
This shouldn't be set in the buildscript, plus it shouldn't be needed in newer snapshots.

Reply:
Alright I'll remove it once I get the chance

Reply:
Fixed it via here.